cluster-operator/src/test/java/io/strimzi/operator/cluster/model/metrics/MetricsModelTest.java

```diff
@@ -40,8 +40,10 @@ public void testMetrics()   {
                     .withConfigMapKeyRef(new ConfigMapKeySelector("my-key", "my-name", false))
                 .endValueFrom()
                 .build();
+        System.out.println("MetricsConfig:" + metricsConfig);
```

Comment:
Don't forget to remove debugging code everywhere.

Reply:
Removed